### PR TITLE
Release note for LegacyServiceAccountTokenNoAutoGeneration

### PR DESCRIPTION
**Pull request by mburke5678**
Release note for LegacyServiceAccountTokenNoAutoGeneration. Slight re-wording at the [request](https://bugzilla.redhat.com/show_bug.cgi?id=2093995#c32) of support

https://github.com/openshift/openshift-docs/pull/47150

Preview: [LegacyServiceAccountTokenNoAutoGeneration is on by default](http://file.rdu.redhat.com/~mburke/BZ-2093995-release-note/release_notes/ocp-4-11-release-notes.html#ocp-4-11-notable-technical-changes)

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -314,13 +314,18 @@ The `haproxy.router.openshift.io/balance` variable, which sets the router load-b
...
 
-However, OpenShift Controller Manager still requires Service Account tokens to appear in secrets in order to function properly, and it will continue to request these tokens in secrets in {product-title} 4.11. In a future release, the `TokenRequest` API will be used instead.
+If you need a service account token secret, you must manually use the TokenRequest API to request bound service account tokens or create a service account token secret. 
 
-Service Account token secrets will still appear as auto-generated in {product-title} 4.11. However, instead of two secrets per Service Account, there will now be only one, which will be further reduced to zero in a future release. For now, dockercfg secrets will still be generated as secrets and no secrets will be deleted during upgrades.
+After upgrading to {product-version}, existing service account token secrets are not deleted and continue to function as expected.
```
Comment (opayne1) [inline release_notes/ocp-4-11-release-notes.adoc:322]: See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update
```suggestion
After updating to {product-version}, existing service account token secrets are not deleted and continue to function as expected.
```

**Review by opayne1 — COMMENTED** [review]
Just a few small nits from me!

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -314,13 +314,18 @@ The `haproxy.router.openshift.io/balance` variable, which sets the router load-b
...
+If you need a service account token secret, you must manually use the TokenRequest API to request bound service account tokens or create a service account token secret. 
 
-Service Account token secrets will still appear as auto-generated in {product-title} 4.11. However, instead of two secrets per Service Account, there will now be only one, which will be further reduced to zero in a future release. For now, dockercfg secrets will still be generated as secrets and no secrets will be deleted during upgrades.
+After updating to {product-version}, existing service account token secrets are not deleted and continue to function as expected.
+
+Service Account token secrets still appear as auto-generated in {product-title} 4.11. However, instead of two secrets per Service Account, there will now be only one, which will be further reduced to zero in a future release. These tokens do not work. For now, dockercfg secrets are still being generated as secrets and no secrets will be deleted during upgrades.
```
Comment (xingxingxia) [inline release_notes/ocp-4-11-release-notes.adoc:324]: > These tokens do not work

This `These tokens do not work` is one of the reported issues described in https://issues.redhat.com/browse/OCPBUGS-13226 . Tracing the source commit, found it is here. @mburke5678 , seems https://github.com/openshift/openshift-docs/pull/47436 did not have QE review. It is better to have had, even if it is `Slight re-wording` :) CC @bergerhoffer

Comment (mburke5678) [reply release_notes/ocp-4-11-release-notes.adoc:324]: @xingxingxia Thank you for catching this.

**Pull request merged**